solutions/basic/stack_definition.json

```diff
@@ -559,15 +592,15 @@
         },
         {
           "name": "ci_code_engine_project",
-          "value": "generative-ai-sample-app-ci-project"
+          "value": "ref:../Workload - Code Engine Projects/outputs/project_1_name"
         },
         {
           "name": "cd_code_engine_project",
-          "value": "generative-ai-sample-app-cd-project"
+          "value": "ref:../Workload - Code Engine Projects/outputs/project_2_name"
         },
         {
           "name": "registry_namespace",
-          "value": "rag-sample-app"
+          "value": "rag-app"
```

Comment:
Is this not a breaking change? Will the old ICR namespace be destroyed and new one created?